packages/qwik/src/core/v2/tests/render-api.spec.tsx

```diff
@@ -46,15 +46,16 @@ const defaultManifest: QwikManifest = {
   manifestHash: 'manifest-hash',
   symbols: {},
   bundles: {},
-  mapping: {},
+  mapping: {
+    click: 'click.js',
+    s_counter: 's_counter.js',
+    s_click: 's_click.js',
+  },
   version: '1',
 };
 
 const defaultCounterManifest: QwikManifest = {
   ...defaultManifest,
-  mapping: {
-    click: 'click.js',
-  },
```

Comment:
If you're moving everything into the default manifest, the countermanifest is no longer useful, so you should remove it.

I think it's fine to do it, the manifest may contain entries that aren't used.